lib/helper/storage.js

```diff
@@ -396,7 +398,7 @@ class Storage {
       self.backup(atom.packages.getActivePackage('ftp-remote-edit').metadata.version);
 
       // migrate
-      let newConfigArray = {};
+      let newConfigArray = [];
```

Comment:
newConfigArray is an object so it should be declared as object and not as an array

let newConfigArray = {};